### PR TITLE
Blocks Page

### PR DESCRIPTION
**Pull request by outragedhuman**
Adds the full page blocks app. 

TODO:

- [x] Final review

### CODE REVIEW

**Review by outragedhuman — COMMENTED** [review]
self review complete

File: src/components/JUPTable/JUPTable.tsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ const TableTitle: React.FC<ITableTitleProps> = ({ title, path, DisplayedComponen
   return (
     <>
       {title && (
-        <TitleText variant="h6" id="tableTitle">
+        <TitleText key={`title-${title}`} variant="h6" id="tableTitle">
```
Comment (vwalsh) [inline src/components/JUPTable/JUPTable.tsx:33]: traditionally we've done something to replace/remove special characters and re-format the key string a little. Eg, replace spaces with underscores.

This isn't a problem imo but is an easy change if you see any value in it.

**Review by vwalsh — COMMENTED** [review]
looks good. see small comment and decide if it's necessary

File: src/components/JUPTable/JUPTable.tsx (hunk 1 above)
Comment (outragedhuman) [reply src/components/JUPTable/JUPTable.tsx]: fixed

**Pull request merged**